WordPress/WordPressUITests/Screens/Me/SupportScreen.swift

```diff
@@ -0,0 +1,45 @@
+import UITestsFoundation
+import XCTest
+
+// This screen object is for the Support section. In the app, it's a modal we can get to from Me > Help & Support, or, when logged out, from Prologue > tap either continue button > Help.
+
+private struct ElementStringIDs {
+    static let closeButton = "close-button"
+    static let helpCenter = "help-center-link-button"
+    static let contact = "contact-support-button"
+    static let myTickets = "my-tickets-button"
+    static let contactEmail = "set-contact-email-button"
+    static let activityLogs = "activity-logs-button"
+}
+
+class SupportScreen: BaseScreen {
+    let closeButton: XCUIElement
+    let helpCenter: XCUIElement
+    let contact: XCUIElement
+    let myTickets: XCUIElement
+    let contactEmail: XCUIElement
+    let activityLogs: XCUIElement
+
+    init() {
+        let app = XCUIApplication()
```

Comment:
Nitpick.

Instantiating `XCUIApplication` this way, in the guts of the type, binds the resulting instance to the value returned by `XCUIApplication`. 

When called with no arguments `XCUIApplication()` generates a proxy class for the app the test target is configured to test.

<img width="1209" alt="Screen Shot 2021-07-12 at 4 07 39 pm" src="https://user-images.githubusercontent.com/1218433/125239034-70c52800-e32b-11eb-8ed5-bcf659af3d43.png">

But, nothing stops us from instantiating a different app using `XCUIApplication(bundleIdentifier:)`. If by any chance we wanted to create this screen object around an app other than the one configured in the target, we wouldn't be able to do it.

Granted, it's unlikely we'd want to do that, because, by their very nature, screen objects are application specific (hence the nitpick). Still, there's a small tweak that we can do to this code that leaves the door open for that use case and makes the type follow the Dependency Injection best practice.

```suggestion
    init(app: XCUIApplication = XCUIApplication()) {
```

Reply:
Thanks for bringing this up! Given that there doesn't seem to be a use case for instantiating a different app, I'll leave it as-is for now. However, if we decide later that we'd be able to use this in shared UI test infrastructure and need to change it, I'll be happy to make the changes.